src/Routers/clients.js

```diff
@@ -0,0 +1,158 @@
+const express = require('express');
+
+const useCasesClients = require('../UseCases/clients');
+
+const router = express.router();
+
+router.get('/clients', async (request, response) => {
+    try{
+        
+        const allClients = await useCasesClients.getClients()
+
+        response.json({
+            success: true,
+            message: 'allClients',
+            data: {
+                clients: allClients,
+            }
+        })
+
+    } catch (error) {
+
+        response.status(400)
+        response.json({
+            success: false,
+            message: 'error at get all clients',
+            error: error.message
+        })
+    }
+})
+
+router.get('/clients/:id', async (request, response)=> {
+    try{
+        const idClient = request.params.id;
+        const clientFound = await useCasesClients.getById(idClient);
+
+        if(!clientFound) throw new Error("Client not found");
+        response.json({
+            success: true,
+            message: "Client found",
+            data: {
+                clients: clientFound
+            }
+        })
+
+    } catch (error) {
+        response.status(404)
+        response.json({
+            success: false,
+            message: "Client not found",
+            error: error.message
+        })
+    }   
+})
+
+router.post('/create', async (request, response) => {
+
+    try{
+        const {body} = request
+        const clientToCreate = request.body
+        const clientCreated = await useCasesClients.create(clientToCreate)
+        const emailStatus = await clientCreated.status 
```

Comment:
@KristomeisterDev Aquí ya no lleva await por que solo estas accediendo a la propiedad del objeto

```suggestion
        const emailStatus = clientCreated.status 
```